helm/values.yaml

```diff
@@ -5,6 +5,9 @@ url: ""
 # If unspecified or empty it will watch all namespaces.
 namespaces: []
 
+# maxRetries -- Maximum retry attempts for failed log sends (logs are discarded after this limit)
```

Comment:
This should mention how long between retries as well.

Reply:
Actually IDK how useful this really is, maybe we should just hardcode a value for now rather than giving an extra config knob.

Something like ~5m should be OK for almost everyone

Reply:
Removed maxRetries from helm/cli args and set max retries to 15.